internal_test_data/network_config_interface_not_defined.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 5
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+user_prefix: 'fruch-testing'
```

Comment:
Looks as debug variant

Reply:
this one is keep been carries away in those unit-tests, it probably can be removed, and the default for those values is good enough for most of the unittests